src/index.js

```diff
@@ -74,7 +74,8 @@ const DATASET_TEMPLATE = {
 };
 
 // localStorage keys
-const KEY_CUSTOM_PRESET  = 'custom-preset',
+const KEY_CUSTOM_GRADS   = 'customgrads',
```

Comment:
Can we change this string to `'custom-grads'` for consistency with the other localStorage key names?